### PR TITLE
Leaderboard: Structure Milestone1

### PR DESCRIPTION
**Pull request by algerina**
In this pull request:
- I created an npm project with Webpack.
-  Added plain HTML markup with minimum styling as required.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1 +1,4 @@
-import _ from 'lodash';
\ No newline at end of file
+import _ from 'lodash'; /* eslint-disable */
+import './style.css';
+
+element.innerHTML = _.join(['Hello', 'webpack'], ' ');
```
Comment (dasileker) [inline src/index.js]: - [x] Nice start but let's make it great, please use the arrow function as the requirements say

llll
.

**Review by dasileker — CHANGES_REQUESTED** [review]
### **Hi @algerina,** 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.js (hunk 1 above)
Comment (algerina) [reply src/index.js]: This is just the basic structure with plain HTML and some CSS, There is no JavaScript functions yet, I will include them in next Milestone.

Comment (dasileker) [reply src/index.js]: True, but the function that you will add it will not work yet.

**Review by carlos-ssh — APPROVED** [review]
# 🍾🍾 **STATUS APPROVED** 👏🏽👏🏽

Hi @algerina , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Approved


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**